table_schema.go

```diff
@@ -56,8 +56,18 @@ func fetchTableSchemas(ctx context.Context, client *spanner.Client) ([]*tableSch
 		FROM INFORMATION_SCHEMA.TABLES AS T
 		LEFT OUTER JOIN FKReferences AS F ON T.TABLE_NAME = F.Referenced
 		WHERE T.TABLE_CATALOG = "" AND T.TABLE_SCHEMA = ""
-		ORDER BY T.TABLE_NAME ASC
-	`))
+`
+	var params map[string]interface{}
+	if len(targetTables) > 0 {
+		sql += " AND T.TABLE_NAME IN UNNEST(@TargetTables) "
+		params = map[string]interface{}{"TargetTables": targetTables}
+	}
+	sql += " ORDER BY T.TABLE_NAME ASC"
```

Comment:
I think it is important for the final query to be predictable for readability. How about filtering the tables in the application layer, not SQL layer?

The limit for [tables per database](https://cloud.google.com/spanner/quotas#tables) is 5,000, but I guess most users don't have such giant table list. So I think it's unlikely to have performance bottleneck even if we perform the filtering in the application layer.

Reply:
Yes, I think it would be better to filter in application layer, so I have fixed it.